src/tools/simeng/main.cc

```diff
@@ -202,10 +202,10 @@ int main(int argc, char** argv) {
   }
 
   // Read the process image and copy to memory
```

Comment:
Reword comment to better suit the new functionality i.e. we're not copying anymore but rather getting the `processMemory` pointer to point to the image vector

Reply:
Comment persist after file change